app/services/message_sender.rb

```diff
@@ -15,12 +16,19 @@ def run
         client.chat_postMessage(
           channel: channel_id,
           as_user: true,
-          text: message.body
+          text: escaped_message_body
         )
       end
     end
   end
 
+  def escape_specific_characters(message_body)
```

Comment:
this might be kind of overkill...but what would you think about making a baby service class for this? That way, it would be easy to unit test